libevmasm/Assembly.h

```diff
@@ -235,6 +235,28 @@ class Assembly
 	/// Returns map from m_subs to an index of subcontainer in the final EOF bytecode
 	std::map<uint16_t, uint16_t> findReferencedContainers() const;
 
+	/// Assembly items types handlers to be used by the dispatching functions for EOF or legacy.
+	void handleOperation(AssemblyItem const& item, bytes& bytecode) const;
+	void handlePush(AssemblyItem const& item, bytes& bytecode) const;
+	void handlePushLibraryAddress(AssemblyItem const& item, LinkerObject& ret) const;
```

Comment:
The `_` prefix is missing in all of the declarations below.

Also, I'd rename `ret` to `_linkerObject`.
```suggestion
	void handlePushLibraryAddress(AssemblyItem const& _item, LinkerObject& _linkerObject) const;
```